demos/noise_suppression_demo/python/noise_suppression_demo.py

```diff
@@ -73,18 +74,25 @@ def main():
 
     log.info('OpenVINO Inference Engine')
     log.info('\tbuild: {}'.format(get_version()))
-    ie = IECore()
+    core = Core()
 
     # read IR
-    log.info("Reading model {}".format(args.model))
-    model_xml = args.model
-    model_bin = model_xml.with_suffix(".bin")
-    ie_encoder = ie.read_network(model=model_xml, weights=model_bin)
+    log.info("Reading model {}".format(args.model_path))
+    model = core.read_model(args.model_path)
 
     # check input and output names
-    input_shapes = {k: v.input_data.shape for k, v in ie_encoder.input_info.items()}
-    input_names = list(ie_encoder.input_info.keys())
-    output_names = list(ie_encoder.outputs.keys())
+    input_shapes = {}
+    input_names = []
+    output_names = []
+    for i in range(len(model.get_parameters())):
+        input_shapes[model.inputs[i].get_any_name()] = model.inputs[i].shape
+        input_names.append(model.inputs[i].get_any_name())
+        output_names.append(model.outputs[i].get_any_name())
+
+    # sorting by names because then we get data from model it is disordered
+    input_names.sort()
+    output_names.sort()
+    input_shapes = dict(sorted(input_shapes.items()))
 
     assert "input" in input_names, "'input' is not presented in model"
```

Comment:
get_any_name returns any of possible name for tensor if it has multiple names, such check is not safe (you can have tensor with this name, but get_any_name return another name)

Reply:
Done